src/front-door/azext_front_door/_params.py

```diff
@@ -122,6 +122,10 @@ def load_arguments(self, _):
             c.argument('http_port', type=int, help='HTTP TCP port number.')
             c.argument('https_port', type=int, help='HTTPS TCP port number.')
             c.argument('weight', type=int, help='Weight of this endpoint for load balancing purposes.')
+            c.argument('private_link_alias', help='The Alias of the Private Link resource. Populating this optional field indicates that this backend is \'Private\'.')
+            c.argument('private_link_resource_id', help='The Resource Id of the Private Link. Populating this optional field indicates that this backend is \'Private\'.')
```

Comment:
What's private link resource id? How can user obtain this value?

Reply:
The private link ID will be shared by the owners of the private link, they're usually an owner of a service that can be privately consumed without exposing that service publicly.

Reply:
Does it mean the private link service id?

Reply:
I'm not sure if Service ID is synonymous to Resource ID -- It's an ARM resource ID (e.g. `/subscriptions/da61bba1-cbd5-438c-a738-c717a6b2d59f/resourceGroups/moeidrg/providers/Microsoft.Network/privateLinkServices/pls-east`)

The naming convention matches up to what the owner of the private link expects to see when they attempt to copy the ID from portal
![image](https://user-images.githubusercontent.com/568368/87110485-019a8c80-c21c-11ea-8eac-d029af47c7a4.png)

Reply:
Private Link is a quite general word. It represent private link service, private endpoint and  private endpoint connection together. Based on the resource id `Microsoft.Network/privateLinkServices`. It should be a ARM resource ID of private link service. Better to add more detail help message.

Reply:
I get your point -- that's good feedback, we'll probably wait to gather more feedback from users re: help text/tips and make the appropriate adjustments in the next API release